lib/news_scraper/trainer/preset_selector.rb

```diff
@@ -27,44 +27,42 @@ def select
         end
         return if selected_option == 'skip'
 
-        selected_index = pattern_options[selected_option]
-        selected_preset_code = transform_results[selected_index].first
-        @data_type_presets[selected_preset_code].merge('variable' => [selected_preset_code, @data_type].join('_'))
+        selected_preset_code = pattern_options[selected_option]
+        result = transform_results[data_type][selected_preset_code].merge(
+          'variable' => [selected_preset_code, data_type].join('_')
+        )
+        result.delete('data')
```

Comment:
Don't want to confuse people, data shouldnt be handled at this point, so it shouldn't return data - just the selected option